vhdbuilder/packer/components.json

```diff
@@ -59,13 +54,11 @@
       "downloadURL": "mcr.microsoft.com/containernetworking/cni-dropgz:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "v0.0.4.1",
-        "v0.0.15",
         "v0.1.1"
```

Comment:
I think v0.1.1 is used only for experimental swift v2 in a few test clusters. Please check with Evan Baker and Matt Long.

Reply:
Have pinged them.